arkouda/extrema.py

```diff
@@ -0,0 +1,83 @@
+from arkouda.client import generic_msg, verbose
```

Comment:
@mcdobe100 do we need this file since you added these to the` pdarrayclass.py` file?

Reply:
Looks like you are right, we do not need this file! I guess that I had a misunderstanding of how the Python side worked, sorry about that. Pushing up the removal of these files now